### ISSUE TITLE
A11y_Manage.IIS.Net_Installation Popup_Keyboard: Links in the agreement are not accessible using keyboard

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/IIS.WebManager#210]
**Environment:**
https://manage.iis.net/connect 
Browser: Microsoft Edge 42.17134.1.0, Google Chrome 69.0.3497.100

**MAS ID Violated:**
MAS 2.1.1

**Repro Steps:**
1. Hit the URL "https://manage.iis.net" 
2. Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file
3. Verify links in the agreement are accessible using keyboard or not

**Actual Result:**
Links in the agreement are not accessible using keyboard

**Expected Result:**
Links in the agreement should be accessible using keyboard

**User Impact:**
Users with disabilities will finds difficult while installing if links are not accessible using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug_210 _links are not accessible using keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2433882/Bug_210._links.are.not.accessible.using.keyboard.pptx)

**Comment by jhkimnew** [comment]
I discussed with @shhsu and we decided not to fix. We compared other setup program's EULA and noticed that there is no clickable link at all and users should copy and paste the link manually as a work-around way, which is applicable to this issue as well.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.1.1; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev3;
